### ISSUE TITLE
no Brandweer events are coming in for a while now

### ISSUE THREAD

**Issue by oklih** [cyberjunky/home-assistant-p2000#58]
Hello,

I am not getting any of the brandweer events in my region. I think this started since the disciplines has change from the old version to the new versions (Brandweerdiensten).
I have configured the code now in noumerous ways, also copied the code from the repository. coding with no Long and Lat. so it would take HA default.
Only when I set it to have no disciplines (aka accept all)  I get the ambulance events from my region, but just them. I kon that there are events there, because I am a volunteer fireman. It is a great component, but if it not working for Brandweer, then it is not so great. :P 

greets Hilko

**Comment by oklih** [comment]
Hello Is there something wrong with the config of mine. Because the logs are searching in ambulancediensten for brandweerdiensten. Maybe I'm reading this wrong. :)

```
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] New P2000 event found: A2 Vlijmen Rit: 137676, at Fri, 17 Dec 2021 15:27:59 +0100
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] Filtering on Regio(s) ['21']
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] Searching for regio 21 in 21
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] Regio matched
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] Filtering on Disciplines(s) ['Brandweerdiensten', ' Politiediensten']
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] Searching for discipline Brandweerdiensten in Ambulancediensten
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] Discipline mismatch, discarding
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] Searching for discipline  Politiediensten in Ambulancediensten
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] Discipline mismatch, discarding
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] New P2000 event found: B2 HZL (D10.1) Neurologie Charlotte Jacobslaan SGRAVH : 15423, at Fri, 17 Dec 2021 15:28:06 +0100
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] Filtering on Regio(s) ['21']
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] Searching for regio 21 in 15
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] Regio mismatch, discarding
```

A piece of the debugging log

**Comment by jan-dewit** [comment]
I see the same logging passing by, probably this has to do with a configuration mismatch? 
For example, this does not look the way like it should in my opinion:
2021-12-17 15:28:12 DEBUG (MainThread) [custom_components.p2000.sensor] Searching for discipline Brandweerdiensten in Ambulancediensten

@cyberjunky would it be possible to share your insights on this topic? Thanks in advance!

**Comment by oklih** [comment]
Hello i have been looking at the config and I don't think anything is wrong. The probleem is that the rss feed by Erwin had no regio 21 Brabant noord any more. Could you ask Erwin why this is? I can't   contact information on the p2000 website.
Doing a search on capcode or regio there is no Brabant Noord
Thanks in advance.

**Comment by ArnolddeBolster** [comment]
@oklih am i right that there's only 1 capcode in Brabant Noord ?
![image](https://github.com/cyberjunky/home-assistant-p2000/assets/71195708/cdddf004-b772-405d-a5ed-5368c1624040)

**Comment by oklih** [comment]
> @oklih am i right that there's only 1 capcode in Brabant Noord ? ![image](https://private-user-images.githubusercontent.com/71195708/321304786-cdddf004-b772-405d-a5ed-5368c1624040.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3Njc1NTQsIm5iZiI6MTcxMjc2NzI1NCwicGF0aCI6Ii83MTE5NTcwOC8zMjEzMDQ3ODYtY2RkZGYwMDQtYjc3Mi00MDVkLWE1ZWQtNTM2OGMxNjI0MDQwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2NDA1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWViOTgxNWFmM2MwNGU0NGM1Njk0Mzc2MGY0NTcwNGRkMzI5ZDk5MjdhNTZjMTU2NTk1ZmRkMmQwZDJmZWJkNDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bqIgzHgimBobX4971ztNcyDXVTFkPeJmsk23Kg8vVqk)

Hello, no there are more then 1 capcode. I think a lot.
This link is just for my hometown Oss.

https://112alarm.net/capcodes.php?regio=21&plaats=OSS&dienst=BRW

**Comment by ArnolddeBolster** [comment]
You're right, I can see them also in 112meldingen.nl so i think the rss feed contains filtered data and now whole of Netherlands.
So maybe for you the RTL stick option is more reliable or try p2000 github option from leeuwte

**Comment by oklih** [comment]
Thank you for the given solutions. I will try this. 👍🏼
